### DISCUSSION TITLE
Harmonize documentation regarding issues/requests/contributions/discussions

### DISCUSSION THREAD

**Discussion by thyttan** [ghostty-org/ghostty#2912] [Feature Requests, Ideas] [answered]
# Description
I found contradicting information when wanting to add [a feature request](https://github.com/ghostty-org/ghostty/discussions/2911). As seen in the two excerpts below.

> Please report any issues you have, including feature requests! Because we're in a closed beta period, there aren't really many rules -- just open 'em up and we'll have a discussion.
>
> That said, feel free to contribute! I would love that. If you want any help, ask in Discord and I'll do my best to point you in the right direction or even pair (time permitting) if you're interested.

*[README_TESTERS.md#reporting-issues-and-contributing](https://github.com/ghostty-org/ghostty/blob/main/README_TESTERS.md#reporting-issues-and-contributing)*

> This document describes the development process for Ghostty. It is intended for anyone considering opening an issue or pull request. If in doubt, please open a discussion; we can always convert that to an issue later.
> (...)
> I'd like to contribute!
>
> All issues are actionable. Pick one and start working on it. Thank you. If you need help or guidance, comment on the issue. Issues that are extra friendly to new contributors are tagged with "contributor friendly".
> 
> I have a bug!
> 
> 1. Search the issue tracker and discussions for similar issues.
> 2. If you don't have steps to reproduce, open a discussion.
> 3. If you have steps to reproduce, open an issue.
> 
> I have an idea for a feature!
> 
> 1. Open a discussion.
> 
> I've implemented a feature!
> 
> 1. If there is an issue for the feature, open a pull request.
> 2. If there is no issue, open a discussion and link to your branch.
> 3. If you want to live dangerously, open a pull request and hope for the best.

*[CONTRIBUTING.md#quick-guide](https://github.com/ghostty-org/ghostty/blob/main/CONTRIBUTING.md#quick-guide)*

## I suggest that:
- README_TESTERS.md is updated with a very short summary of the information from CONTRIBUTING.md together with a forwarding link to the same.
- the README.md is updated with a link forwarding to CONTRIBUTING.md in a suitable place - maybe as one of the links at the top?

I can submit a PR with changes if wanted.

**Comment by mitchellh** [comment, marked as answer]
In this case, `README_TESTERS.md` is outdated. I'd accept a PR to rewrite it. It's not a _huge_ priority because that whole file is going away when Ghostty is public anyways. 😄
